source/documentation/internal/guest-agent/guest-agent.html.md

```diff
@@ -19,38 +19,38 @@ It provides:
-      * `**`Logged` `in` `users`**` - List of all logged-in users.
-      * `**`Active` `user`**` - The user which currently is using the virtual machine's "physical hardware". Redundant since RHEV-M is treating the SPICE's ticket owner as the active user.
-      * `**`Disks` `Usage`**` - VM Disks utilization.
-      * `**`network-interfaces`**` - Internal guest network mapping (MAC, name, ipv4, ipv6).
-      * `**`FQDN`**` - Reports the configured (F)ully (Q)ualified (D)omain (N)ame of the guest os
+* **Machine name** -- Show the virtual machine's host name.
+* **Operating system version** -- Show the operating system's version. Linux: this value is the kernel version. Windows: it is the Windows version name (e.g. Windows XP or Windows 7).
+* **Installed applications** -- List in installed applications. Linux: application list is set using the configuration file. Windows: installed applications list is based on value read from registry.
+* **Available RAM** -- The amount of unused physical memory. This value probably include memory like cache, or else the memory usage will always be (or near) 100% usage.
+* **Logged in users** -- List of all logged-in users.
+* **Active user** -- The user which currently is using the virtual machine's "physical hardware". Redundant since RHEV-M is treating the SPICE's ticket owner as the active user.
+* **Disks usage** -- VM Disks utilization.
+* **Network interfaces** -- Internal guest network mapping (MAC, name, ipv4, ipv6).
+* **FQDN** -- Reports the configured (F)ully (Q)ualified (D)omain (N)ame of the guest os
 
 ## Notifications
 
 The agent notify on the following events:
 
-      * `**`Power` `Up`**` - Send when agent start its execution.
-      * `**`Power` `Down`**` - unused
-      * `**`Heartbeat`**` - Message is send every few second to notify that the agent is running. The notification includes the guest's available RAM.
-      * `**`User` `Info`**` - Active user was changed.
-      * `**`Session` `Lock`**` - Desktop was locked (Windows).
-      * `**`Session` `Unlock`**` - Desktop was unlocked (Windows).
-      * `**`Session` `Logoff`**` - A user was logged off (Windows).
-      * `**`Session` `Logon`**` - A user was logged on (Windows).
-      * `**`Agent` `Uninstalled`**` - Agent was removed from system. Expected to be send from the agent's installer.
+* **Power up** -- Send when agent start its execution.
+* **Power down** -- unused
+* **Heartbeat** -- Message is send every few second to notify that the agent is running. The notification includes the guest's available RAM.
+* **User info** -- Active user was changed.
+* **Session lock** -- Desktop was locked (Windows).
+* **Session unlock** -- Desktop was unlocked (Windows).
+* **Session logoff** -- A user was logged off (Windows).
+* **Session logon** -- A user was logged on (Windows).
+* **Agent uninstalled** -- Agent was removed from system. Expected to be send from the agent's installer.
```

Comment:
Suggested edit (changes in **bold**)
* **Agent uninstalled** -- Agent was removed from **the** system. Expected to be **sent** from the agent's installer.